appendix-basic65.tex

```diff
@@ -2166,16 +2285,26 @@ \subsection{COLOR}
 \index{BASIC 65 Commands!COLOR}
 \begin{description}[leftmargin=2cm,style=nextline]
 \item [Token:] \$E7
-\item [Format:] {\bf COLOR} colour-index
-\item [Usage:] The command works in the same way as {\bf FOREGROUND},
-               i.e: sets the foreground colour
-               (text colour) of the screen to the colour argument, which
-                must be in the range of 0 to 31. Refer to the table
-               under {\bf BACKGROUND} on page \pageref{colourtable} for
-               the colour values and their corresponding colours.
+\item [Format:] {\bf COLOR} colour
+\item [Usage:] Sets the foreground text colour for subsequent {\bf PRINT} commands.
+
+               {\bf colour} the palette entry number, in the range 0 -- 255
```

Comment:
I think the true range of this is still 0-31 (don't think text modes provided by BASIC can display more than 32 colours)

Reply:
Confirmed, thanks. Fixed.